source/data-api.txt

```diff
@@ -53,10 +53,31 @@ The Data API supports two types of endpoints:
 - **Custom Endpoints** are app-specific API routes handled by functions
   that you write. You can use custom endpoints to run your app's backend
   logic or as webhooks that integrate with external services.
-  
+
   To learn more, including how to create and call endpoints, see
   :ref:`Custom HTTPS Endpoints <custom-endpoints>`.
 
+.. _data-api-supported-environments:
+
+Supported Environments
+----------------------
+
+You can call the Data API from most HTTP clients. Supported environments include:
```

Comment:
Can we mention that it's specifically not supported by any client that enforces CORS (with a link to MDN)